README.md

```diff
@@ -1,22 +1,34 @@
+# CadenceRadio
 ![GitHub release (latest by date)](https://img.shields.io/github/v/release/kenellorando/cadence?style=flat-square)
-![GitHub issues](https://img.shields.io/github/issues/kenellorando/cadence?style=flat-square)
 
-# CadenceRadio
+**Cadence** is a fully-featured HTTP API web radio software suite.
+
+It ships everything you need to start a web radio station, including an API server (with search, request, now playing, artwork, and stream information functions), an interactive browser UI, an autopopulating music-metadata database, and custom built-in _Icecast_ and _Liquidsoap_ integration.
+
+Cadence is essentially a metadata broker between your music files and stream services, with a programmatic interface for your audiences. The project ships all components pre-configured to work with each each other so there is hardly any configuration to do.
 
-**Cadence** is a web radio API server. It prepares and communicates data between independent stream software _Icecast_, audio client _Liquidsoap_, and a _Postgres_ database, allowing radio administrators to broadcast audio on the filesystem to the internet while allowing users to search for music and make song requests through a custom web frontend.
+To configure Cadence, set a target directory containing your music, set a few service passwords and hostnames, and you're good to go! Cadence has container releases for amd64 and armv7. All components are containerized, and a Compose file sets the entire radio stack up in minutes. See a live demo on [cadenceradio.com](https://cadenceradio.com/)!
 
-In the current iteration (4.x) of Cadence, no frontend framework is employed and the backend is written in Go. For questions and comments, you are welcome to [open a discussion](https://github.com/kenellorando/cadence/discussions) on Github.
+![cadence5 architecture](https://user-images.githubusercontent.com/17265041/185465196-66fc2249-e43a-46f7-a12f-dbde9aaf8172.png)
 
-Check it out in action: [https://cadenceradio.com/](https://cadenceradio.com/).
+## Get Started
 
-## Feature Architecture
+> **Warning**: The way this repo currently handles configuration involves adding passwords to files which you may accidentally commit, so be careful.
```

Comment:
Should we look into something like what I do for CadenceBot (a comprehensive default-config and an optional, gitignored, overriding config for users to enter these things into)?

Reply:
Yes, that sounds good. Maybe we can create these default `.example` files and the user becomes required to copy it without the `.example` to make it become live, which is gitignored. Just keep in mind that these new Dockerfiles for Liquidsoap and Icecast will expect an exact file name.